src/ai_migrate/tools/kotlin_symbols.py

```diff
@@ -120,5 +129,8 @@ def scan_repository(root_dir: str | None = None) -> dict[str, str]:
     return symbol_tree
 
 
+symbol_lookup_tool = Tool(get_symbol_definition)
```

Comment:
can't we use the annotation here?

Reply:
the annotation in pydantic-ai is a member of pydantic_ai.Agent, so not really